### ISSUE TITLE
Tests from sdist fail

### ISSUE THREAD

**Issue by jayvdb** [thibaudcolas/curlylint#49]
When fetching the sdist from PyPI, the tests fail due to missing resources.

```
[   38s] _________ ERROR collecting curlylint/rules/aria_role/aria_role_test.py _________
[   38s] curlylint/rules/aria_role/aria_role_test.py:9: in <module>
[   38s]     class TestRule(unittest.TestCase, metaclass=RulesTestMeta):
[   38s] curlylint/rules/rule_test_case.py:23: in __new__
[   38s]     fixtures = json.loads(open(tests["fixtures"], "r").read())
[   38s] E   FileNotFoundError: [Errno 2] No such file or directory: '/home/abuild/rpmbuild/BUILD/curlylint-0.12.0/curlylint/rules/aria_role/aria_role_test.json'
[   38s] _ ERROR collecting curlylint/rules/django_forms_rendering/django_forms_rendering_test.py _
[   38s] curlylint/rules/django_forms_rendering/django_forms_rendering_test.py:8: in <module>
[   38s]     class TestRule(unittest.TestCase, metaclass=RulesTestMeta):
[   38s] curlylint/rules/rule_test_case.py:23: in __new__
[   38s]     fixtures = json.loads(open(tests["fixtures"], "r").read())
[   38s] E   FileNotFoundError: [Errno 2] No such file or directory: '/home/abuild/rpmbuild/BUILD/curlylint-0.12.0/curlylint/rules/django_forms_rendering/django_forms_rendering_test.json'
[   38s] _____ ERROR collecting curlylint/rules/html_has_lang/html_has_lang_test.py _____
[   38s] curlylint/rules/html_has_lang/html_has_lang_test.py:9: in <module>
[   38s]     class TestRule(unittest.TestCase, metaclass=RulesTestMeta):
[   38s] curlylint/rules/rule_test_case.py:23: in __new__
[   38s]     fixtures = json.loads(open(tests["fixtures"], "r").read())
[   38s] E   FileNotFoundError: [Errno 2] No such file or directory: '/home/abuild/rpmbuild/BUILD/curlylint-0.12.0/curlylint/rules/html_has_lang/html_has_lang_test.json'
[   38s] _________ ERROR collecting curlylint/rules/image_alt/image_alt_test.py _________
[   38s] curlylint/rules/image_alt/image_alt_test.py:9: in <module>
[   38s]     class TestRule(unittest.TestCase, metaclass=RulesTestMeta):
[   38s] curlylint/rules/rule_test_case.py:23: in __new__
[   38s]     fixtures = json.loads(open(tests["fixtures"], "r").read())
[   38s] E   FileNotFoundError: [Errno 2] No such file or directory: '/home/abuild/rpmbuild/BUILD/curlylint-0.12.0/curlylint/rules/image_alt/image_alt_test.json'
[   38s] _____ ERROR collecting curlylint/rules/meta_viewport/meta_viewport_test.py _____
[   38s] curlylint/rules/meta_viewport/meta_viewport_test.py:9: in <module>
[   38s]     class TestRule(unittest.TestCase, metaclass=RulesTestMeta):
[   38s] curlylint/rules/rule_test_case.py:23: in __new__
[   38s]     fixtures = json.loads(open(tests["fixtures"], "r").read())
[   38s] E   FileNotFoundError: [Errno 2] No such file or directory: '/home/abuild/rpmbuild/BUILD/curlylint-0.12.0/curlylint/rules/meta_viewport/meta_viewport_test.json'
[   38s] ______ ERROR collecting curlylint/rules/no_autofocus/no_autofocus_test.py ______
[   38s] curlylint/rules/no_autofocus/no_autofocus_test.py:9: in <module>
[   38s]     class TestRule(unittest.TestCase, metaclass=RulesTestMeta):
[   38s] curlylint/rules/rule_test_case.py:23: in __new__
[   38s]     fixtures = json.loads(open(tests["fixtures"], "r").read())
[   38s] E   FileNotFoundError: [Errno 2] No such file or directory: '/home/abuild/rpmbuild/BUILD/curlylint-0.12.0/curlylint/rules/no_autofocus/no_autofocus_test.json'
[   38s] _ ERROR collecting curlylint/rules/tabindex_no_positive/tabindex_no_positive_test.py _
[   38s] curlylint/rules/tabindex_no_positive/tabindex_no_positive_test.py:9: in <module>
[   38s]     class TestRule(unittest.TestCase, metaclass=RulesTestMeta):
[   38s] curlylint/rules/rule_test_case.py:23: in __new__
[   38s]     fixtures = json.loads(open(tests["fixtures"], "r").read())
[   38s] E   FileNotFoundError: [Errno 2] No such file or directory: '/home/abuild/rpmbuild/BUILD/curlylint-0.12.0/curlylint/rules/tabindex_no_positive/tabindex_no_positive_test.json'
```

**Comment by thibaudcolas** [comment]
Hey @jayvdb, thank you for the report. Generally I prefer to not even publish tests in packages I maintain, so they are faster to install, although I imagine it’s fine for the `sdist` to contain them as long as the wheel doesn’t.

Can you explain what use case you have to run the tests from the sdist though?

**Comment by jayvdb** [comment]
I am trying to package curlylint for openSUSE.  https://build.opensuse.org/package/show/home:jayvdb:py-new/python-curlylint .  I have commented out the tests in the .spec at the moment, but they need to be enabled before I can submit the package into the main openSUSE package collection.

The PR I have set up only adds to the sdist, not the wheel.

**Comment by thibaudcolas** [comment]
Thank you for the info! I’ve just released [v0.12.1 on PyPI](https://pypi.org/project/curlylint] with this fix.
